.azure/pipelines/ci.yml

```diff
@@ -577,9 +576,10 @@ stages:
       agentOs: Windows
       timeoutInMinutes: 180
       steps:
-      - script: .\restore.cmd -ci
-        displayName: Restore
-      - script: .\build.cmd -ci -NoRestore -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
+      # Build the shared framework
+      - script: ./build.cmd -ci -all -pack
+        displayName: Build shared fx
+      - script: .\build.cmd -ci -test -projects eng\helix\helix.proj /p:IsRequiredCheck=true /p:IsHelixJob=true /p:BuildAllProjects=true /p:BuildNative=true /p:ASPNETCORE_TEST_LOG_DIR=artifacts/log -bl
```

Comment:
This will repeat lots of the above build step. Suggest adding ` -noBuildRepoTasks -noRestore -noBuild`.